packages/typed/src/generate-schema.ts

```diff
@@ -115,7 +120,7 @@ const transformCaptureTypeArguments = <T extends ts.Node>(
       return node;
     }
     const tp = node as ts.TupleTypeNode;
-    const et: ts.NodeArray<ts.TypeNode> = tp.elementTypes.map(e => {
+    const et: ts.NodeArray<ts.TypeNode> = tp.elements.map(e => {
```

Comment:
It's TypeScript AST breaking change. (It might be related to variadic tuple type feature...?)

- `ts <= 3.9`: tuple type node has `elementTypes` children.
- `ts >= 4.0`: tuple type node has `elements` children.

See also https://ts-ast-viewer.com/#code/C4TwDgpgBAHlC8UDaBnYAnAlgOwOYF0g .